src/vis/vis.js

```diff
@@ -1152,6 +1152,32 @@ var Vis = cdb.core.View.extend({
         }
       });
     }
+
+    var self = this; 
+
+    if (layerView.tooltip){
+      
+      layerView.bind('mouseover', function() {
+        
+        self.mapView.setCursor('nw-resize'); 
+      });
+      layerView.bind('mouseout', function() {
+        
+        self.mapView.setCursor('auto'); 
+      }); 
+    }
+
+    if (layerView.infowindow){
```

Comment:
Also, we already have mouse bindings here: https://github.com/CartoDB/cartodb.js/blob/228-change_pointer_tooltip/src/vis/vis.js#L1276 https://github.com/CartoDB/cartodb.js/blob/228-change_pointer_tooltip/src/vis/vis.js#L1280… why the repetition?